scripts/update_changelog.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+#
+# Run this script from the top level directory of the trillian repo e.g.
+# with scripts/update_changelog.sh.
+#
+# GOPATH must be set.
+
+set +e
+d=${GOPATH[0]}
+PATH=$PATH:${d}/bin
+
```

Comment:
Maybe add "TODO(Martin2112): shift to official repo if/when https://github.com/aktau/github-release/pull/81 is merged"

Reply:
Done.